home/javascript/react/containers/FigurePhenotypeTable.js

```diff
@@ -63,7 +67,7 @@ const FigurePhenotypeTable = ({url, hideFigureColumn = false, navigationCounter,
         <DataTable
             columns={columns}
             dataUrl={url}
-            rowKey={row => row.rowKey}
+            rowKey={() => Math}
```

Comment:
Is this line supposed to include .random()? I'm still not sure the reason why random is better than a unique id